### PR TITLE
Interactive-to-do-list branch PR#4

### PR DESCRIPTION
**Pull request by efmomoh**
This time, I : 

Adds a new javascript function file to bundle the webpack
U see ES6 syntaxes
Store list to local storage

### CODE REVIEW

File: src/update.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+export const initStatusUpdates = (
+  tasksArray,
+  saveToLocalStorage,
+  clearListButton,
+  updateIndexes,
+  performJob,
+  saveTasksToLocalStorage
+) => {
+  const checkboxes = document.querySelectorAll("input[type='checkbox']");
+  console.log(checkboxes);
```
Comment (codepantha) [inline src/update.js]: - [ ] Please remove console.logs from your codebase as it is considered best practice to do so in order to reduce code clutter and improve readability, maintainability and even performace in cases where `console.log` is used within loops or frequently executed functions

File: src/update.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    tasksArray = tasksArray.filter((task) => !task.completed);
+    saveToLocalStorage();
+  });
+
+  const savedTasks = JSON.parse(localStorage.getItem("tasks"));
+  tasksArray = savedTasks || [];
+
+  updateIndexes();
+  performJob();
+};
```
Comment (codepantha) [inline src/update.js]: - [ ] You have done a great job storing the to-do tasks in `localStorage` 👏 However, when the page gets reloaded, the `to-do` application doesn't show the list of `to-do`s in `localStorage`. Please fix this issue 🙏 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @efmomoh 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Tasks completion by selecting the corresponding checkbox 📌 
✔️ The "clear all completed" functionality is implemented 📌 
✔️ No linter errors 🟢 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/update.js (hunk 1 above)
Comment (peterrobert) [reply src/update.js]: This change has been done well

File: src/update.js (hunk 2 above)
Comment (peterrobert) [reply src/update.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @efmomoh, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️
[ ] Great work on  your design ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**